meson.build

```diff
@@ -26,6 +26,7 @@ rt = cc.find_library('rt')
 pipewire = dependency('libpipewire-0.3', version: '>= 0.2.9')
 wayland_client = dependency('wayland-client')
 wayland_protos = dependency('wayland-protocols', version: '>=1.14')
+drm = dependency('libdrm')
```

Comment:
Can we use a `partial_dependency`, since we only rely on the header constants?

Reply:
ok, looking at it (sorry, I don't know meson very well / at all)

Reply:
is `drm = dependency('libdrm').partial_dependency(includes: true)` ok ?
should I add an other `partial_drm` dependency ?

Reply:
This should be fine, yes.